bundle/internal/tf/schema/resources.go

```diff
@@ -52,6 +52,7 @@ type Resources struct {
 	Pipeline                 map[string]*ResourcePipeline                 `json:"databricks_pipeline,omitempty"`
 	Provider                 map[string]*ResourceProvider                 `json:"databricks_provider,omitempty"`
 	Recipient                map[string]*ResourceRecipient                `json:"databricks_recipient,omitempty"`
+	RegisteredModel          map[string]*ResourceRegisteredModel          `json:"databricks_registered_model,omitempty"`
```

Comment:
Temporarily modified this file while waiting for https://github.com/databricks/terraform-provider-databricks/pull/2771 to merge and the new provider version to be released.